src/parsers/pm2ps_parser.jl

```diff
@@ -223,21 +228,22 @@ function make_thermal_gen(gen_name, d, bus)
         fixedcost = cost[1][2]
     elseif model == POLYNOMIAL::GeneratorCostModel
         if d["ncost"] == 0
-            cost = x-> 0
+            cost = 0
+            fixedcost = 0.0
         elseif d["ncost"] == 1
-            cost = x-> d["cost"][1]
+            cost = (0.0, 0.0)
+            fixedcost = d["cost"][1]
         elseif d["ncost"] == 2
-            cost = x-> d["cost"][1]*x + d["cost"][2]
+            cost = (0.0, d["cost"][1])
+            fixedcost = d["cost"][2]
         elseif d["ncost"] == 3
-            cost = x-> d["cost"][1]*x^2 + d["cost"][2]*x + d["cost"][3]
+            cost = (d["cost"][1], d["cost"][2]) 
+            fixedcost = d["cost"][3]
         elseif d["ncost"] == 4
-            cost = x-> d["cost"][1]*x^3 + d["cost"][2]*x^2 + d["cost"][3]*x + d["cost"][4]
+            @error("PowerSystems currently can only support polynomial functions upto second degree")
```

Comment:
Should this throw an error like in the else clause?

Reply:
fixed via 460e04f